src/material/core/tokens/m2/mdc/_elevated-card.scss

```diff
@@ -0,0 +1,82 @@
+@use 'sass:map';
+@use '../../../theming/theming';
+@use '../../token-utils';
+
+// The prefix used to generate the fully qualified name for tokens in this file.
+$prefix: (mdc, elevated-card);
+
+// Tokens that can't be configured through Angular Material's current theming API,
+// but may be in a future version of the theming API.
+//
+// Tokens that are available in MDC, but not used in Angular Material should be mapped to `null`.
+// `null` indicates that we are intentionally choosing not to emit a slot or value for the token in
+// our CSS.
+@function get-unthemable-tokens() {
+  @return (
+    // The border-radius of the card.
+      container-shape: 4px,
+    // =============================================================================================
+    // = TOKENS NOT USED IN ANGULAR MATERIAL                                                       =
+    // =============================================================================================
+    // TODO(mmalerba): The elevation tokens don't seem to work currently.
+    //  Should work with MDC to figure out what's wrong.
+      container-elevation: null,
+      container-shadow-color: null,
+    // Angular Material's card is not an interactive element, and therefore does not support states.
+      disabled-container-color: null,
+      disabled-container-elevation: null,
+      disabled-container-opacity: null,
+      dragged-container-elevation: null,
+      dragged-state-layer-color: null,
+      dragged-state-layer-opacity: null,
+      focus-container-elevation: null,
+      focus-state-layer-color:null,
+      focus-state-layer-opacity: null,
+      hover-container-elevation: null,
+      hover-state-layer-color: null,
+      hover-state-layer-opacity: null,
+      pressed-container-elevation: null,
+      pressed-state-layer-color: null,
+      pressed-state-layer-opacity: null,
+    // Angular Material does not currently support surface tint.
+      container-surface-tint-layer-color: null,
+    // MDC does not seem to use these tokens.
+      icon-color: null,
+      icon-size: null,
+  );
+}
+
+// Tokens that can be configured through Angular Material's color theming API.
+@function get-color-tokens($config) {
+  $background: map.get($config, background);
+
+  @return (
+    // The background color of the card.
+      container-color: theming.get-color-from-palette($background, card),
+  );
+}
+
+// Tokens that can be configured through Angular Material's typography theming API.
+@function get-typography-tokens($config) {
+  @return ();
+}
+
+// Tokens that can be configured through Angular Material's density theming API.
+@function get-density-tokens($config) {
+  @return ();
+}
+
+// Combines the tokens generated by the above functions into a single map with placeholder values.
+// This is used to create token slots.
+@function get-token-slots() {
+  @return map.merge(
```

Comment:
All this merging is a little hard to read. Would it make sense to move parts of it out into a local variable?

Reply:
I can simplify this, I think if we had a generic merge-all function for maps, it would make this a lot more clear